### PR TITLE
fix: Temporary default to Axelar USDC tokens on deposit while CCTP is down

### PR DESCRIPTION
**Pull request by rosepuppy**

### CODE REVIEW

File: src/views/forms/AccountManagementForms/DepositForm.tsx [hunk 1]
```diff
@@ -130,6 +131,27 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
     };
   }, []);
 
+  // Temporary default to Axelar USDC tokens while CCTP is down.
+  // Revert this change once CCTP is back up.
+  useEffect(() => {
+    const USDCTokenAddresses = [
```
Comment (yogurtandjam) [inline src/views/forms/AccountManagementForms/DepositForm.tsx:137]: [nit] if we use a set we get constant time lookups instead of having to use `includes`

**Review by yogurtandjam — APPROVED** [review]

**Comment by yogurtandjam** [comment]
[q] do we only need to do this for deposits? i'm guessing this is related to [the slack thread](https://dydx-team.slack.com/archives/C03GKH23YAZ/p1715271985882129)

File: src/views/forms/AccountManagementForms/DepositForm.tsx (hunk 1 above)
Comment (rosepuppy) [reply src/views/forms/AccountManagementForms/DepositForm.tsx]: good call!

**Comment by rosepuppy** [comment]
> [q] do we only need to do this for deposits? i'm guessing this is related to [the slack thread](https://dydx-team.slack.com/archives/C03GKH23YAZ/p1715271985882129)

My understanding is we mainly care about reducing the deposit friction here, and I wanted to minimize temp code as well.

**Pull request merged**